resources/views/pages/user-profile.blade.php

```diff
@@ -65,7 +65,8 @@ class="rounded-circle img-fluid" style="width: 150px;">
                             <form action="{{ route('update.profile') }}" method="post">
                                 @csrf
                                 <div class="row">
-                                    <label for="" class="col-sm-3">{{ __('static.name') }}</label>
+                                    <label data-error="wrong" data-success="right"
+                                        class="col-sm-3">{{ __('static.name') }}</label>
```

Comment:
what is these changes? are these related to validation ? the changes on the profile page is irrelevant to this ticket by the way